workspace.code-workspace

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Can you drop the vscode config file?  Might be worth adding it to the .gitignore

Reply:
Yes. Sorry for that. Would you like me to add "*.code-workspace" to .gitignore in this PR as well?

Reply:
@loopodoopo Would be better to remove the file only in this PR , and create a new PR to change the gitfile. Just for readability. Thx

Reply:
Sure, will do!